projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss

```diff
@@ -138,11 +138,26 @@
     @if ($_variant == 'indigo') {
         @include igx-calendar-typography($type-scale);
 
-        @include igx-card-typography($type-scale, $categories: (
-            title: 'h5',
-            title-small: 'subtitle-2',
-            subtitle: 'subtitle-2',
-            content: 'body-2',
+        @include igx-checkbox-typography($type-scale, $categories: (
+            label: 'body-1',
+        ));
+
+        @include igx-input-group-typography($type-scale, $categories: (
+            input-text: 'body-1'
+        ));
+
+        @include igx-calendar-typography($type-scale, $categories: (
+            header-year: 'overline',
```

Comment:
Rather than doing it here which is correct, why don't we do it on the base theme i.e. material and here it will be inherited from there?

Reply:
Sure, I will do it in material.

Reply:
not sure if this will be considered a breaking change visually but we may have to note it down in release notes @simeonoff